openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/TargetPacketItemChooser.vue

```diff
@@ -21,9 +21,10 @@
 -->
 
 <template>
-  <v-container class="c-chooser px-0">
+  <!-- tgt-pkt-item-chooser class used by Graph.vue to size the graph -->
+  <div class="pt-4 tgt-pkt-item-chooser">
```

Comment:
This class is basically only used by Graph to know how much height the TargetPacketItemChooser takes up